live-build/misc/ansible-roles/appliance-build.dcenter/tasks/main.yml

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- include: ldap.yml
```

Comment:
LDAP support is something that may be generally useful to have in the `dev` variant. We currently have it in the `dlpx-trunk` images based on illumos. I don't know that we've intentionally decided _not_ to have it in the Linux `dev` variant, just that it hasn't been implemented yet. I'd suggest the team come to a decision about whether or not we want to include LDAP support in the `dev` variant. Key to this discussion is the question of whether we can grant `sudo` permissions to LDAP users while not granting it to non-LDAP users (something that wasn't possible in illumos). LDAP support without `sudo` support is relatively useless for end users, but giving all users `sudo` access is dangerous in that it is an unsupported product configuration (though the risk here might be somewhat mitigated by the existence of the new `qa` variant). If we decide that we do want LDAP support in the `dev` variant, the LDAP logic probably be better split out into a separate Ansible role rather than being part of the DCenter Ansible role.